src/components/Experience.js

```diff
@@ -1,35 +1,131 @@
+import { format } from 'date-fns/esm';
+import { useState, Fragment } from 'react';
 import styles from './experience.module.css';
 
-function handleClick(e) {}
+export default function Experience({ data, onChange, onClick }) {
+  const [isEditing, setIsEditing] = useState(false);
+
+  function handleClick() {
+    isEditing ? setIsEditing(false) : setIsEditing(true);
+  }
+
+  function generateContent(schoolData) {
+    let content = (
```

Comment:
same as in above Component :D Re-check all the stuff from there in here and other occurences that might happen.